source/Halibut.Tests/Support/HalibutRuntimeBuilderExtensionMethods.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Halibut.Tests.Support
+{
+    public static class HalibutRuntimeBuilderExtensionMethods
+    {
+        public static HalibutRuntimeBuilder WithAsyncHalibutFeatureEnabledIfForcingAsync(this HalibutRuntimeBuilder halibutRuntimeBuilder, ForceClientProxyType? forceClientProxyType)
```

Comment:
I was going down a path that needed this a many times, I think it is still worth keeping even though it is called only a couple of times. Since it keeps our big builders smaller.